### PR TITLE
integrating hCaptcha and ReCaptcha adapters

### PR DESCRIPTION
**Pull request by Vahram1995**

### CODE REVIEW

File: src/Libraries/Captcha/Adapters/HcaptchaAdapter.php [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+        } elseif (strtolower($this->type) == 'invisible') {
+            $captchaElement = $this->getInvisibleElement($formIdentifier);
+        }
+        return $captchaElement;
+    }
+
+    /**
+     * Render js source
+     *
+     * @param null   $lang
```
Comment (armanist) [inline src/Libraries/Captcha/Adapters/HcaptchaAdapter.php:89]: Param type is defined as `null`

File: src/Libraries/Captcha/Adapters/RecaptchaAdapter.php [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+     * @var array
+     */
+    protected $errorCodes = array();
+
+
+    /**
+     * RecaptchaAdapter
+     *
+     * @param array $params
+     * @return void
```
Comment (armanist) [inline src/Libraries/Captcha/Adapters/RecaptchaAdapter.php:89]: Remove all return types if they are void

File: src/Libraries/Captcha/Adapters/RecaptchaAdapter.php [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+
+        return $response['success'];
+    }
+
+    /**
+     * Returns the errors encountered
+     *
+     * @return array Errors code and name
+     */
+    public function getErrorCodes(): array
```
Comment (armanist) [inline src/Libraries/Captcha/Adapters/RecaptchaAdapter.php:190]: Where we are calling this method and why `Hcaptcha` doesn't have it?

**Review by hatik92 — APPROVED** [review]

**Review by armanist — COMMENTED** [review]
Fix the conflict in composer.json

**Pull request merged**